frontend/src/app/components/block/block.component.ts

```diff
@@ -58,7 +58,7 @@ export class BlockComponent implements OnInit, OnDestroy {
   overviewError: any = null;
   webGlEnabled = true;
   indexingAvailable = false;
-  auditModeEnabled: boolean = !this.stateService.hideAudit.value;
+  auditModeEnabled: boolean = this.stateService.env.AUDIT && !this.stateService.hideAudit.value;
```

Comment:
@mononaut I think this could be split into two variables, `auditModeEnabled` and `auditModeDisplayed` (or `shouldDisplayAuditMode`), where `auditModeEnabled` is a shortcut for `this.stateService.env.AUDIT` so you don't repeat that throughout the code and `auditModeDisplayed` controls whether it's shown or not, as you have now before this change.